submissions/fabunmiTolulope/phpDatabase/customersOrders.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+include('connection.php');
+ if( isset($_GET['id']) && strlen($_GET['id']) > 0 ) {
+ $id =  $_GET['id'];
+}
+
+try{
```

Comment:
DOn't place your query execution and table inside the same try block. Doing it this way, your query will always be executed. Your query should only run ones as long as cookie has not expired

Reply:
@adaugochi, Thanks for the correction I have tried using different try blocks.